docs/ide-usage.md

```diff
@@ -64,9 +64,9 @@ The following plugins are needed:
      Configure for all the languages used in TEAMMATES:
      * Java: `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`.
      * JavaScript: `JavaScript → Code Style → Formatter → Edit → Tab policy → Spaces only`.
-     * HTML: `Web → HTML Files → Editor → Indent using spaces`.
-     * CSS: `Web → CSS Files → Editor → Indent using spaces`.
-     * XML: `XML → XML Files → Editor → Indent using spaces`.
+     * HTML: `Eclipse → Preferences → Web → HTML Files → Editor → Indent using spaces`.
+     * CSS: `Eclipse → Preferences → Web → CSS Files → Editor → Indent using spaces`.
+     * XML: `Eclipse → Preferences → XML → XML Files → Editor → Indent using spaces`.
```

Comment:
You may want to edit line 46, to mention the exception about html, css, xml indentation. For this section, you may want to add a line above these 3 lines, mentioning the relevant place to change the settings are under `Eclipse -> Preferences`, and remove the `Eclipse -> Preferences` in these 3 lines instead.